File: common/node_common/json_manipulate.js
// ---------------------------------------------------------------
//	json_manipulate.js
//
//					Dec/26/2016
//
// ---------------------------------------------------------------
var text_manipulate=require ('./text_manipulate')
// ---------------------------------------------------------------
exports.json_update_proc = function (json_str,population_in)
{
	const data = JSON.parse (json_str)

	var unit_aa = {}
	unit_aa['name'] = data.name
	unit_aa['population'] = population_in
	unit_aa['date_mod'] = text_manipulate.get_current_date_proc ()

	const json_out = JSON.stringify (unit_aa)

	return	json_out
}

// ---------------------------------------------------------------
exports.json_display_proc = function (key,json_str)
{
	var out_str = key + "\t"
	const data = JSON.parse (json_str)

	out_str  += data.name + "\t"
	out_str += data.population + "\t"
	out_str += data.date_mod

	console.log (out_str)
}

// ---------------------------------------------------------------

File: common/lua_common/json_manipulate.lua
-- ----------------------------------------------------
--	json_manipulate.lua
--
--					Apr/25/2011
-- ----------------------------------------------------
function json_record_out_proc (_index)
	out_str = _index .. "\t"
	out_str = out_str .. result[_index].name .. "\t"
	out_str = out_str .. result[_index].population .. "\t"
	out_str = out_str .. result[_index].date_mod
	print (out_str)
end
-- ----------------------------------------------------
function json_disp_proc (json_str)
	json = require("json")
	result = json.decode(json_str)
--	table.foreach(result,print)
	table.foreach(result,json_record_out_proc )
end
-- ----------------------------------------------------
